models/badges.js

```diff
@@ -27,6 +28,34 @@ const fetchBadges = async ({
   }
 }
 
+/**
+ * Fetches the data about user badges
+ * @param query { Object }: Filter for badges data
+ * @return {Promise<userBadgeModel|Array>}
+ */
+
+const fetchUserBadges = async (username) => {
+  try {
+    const result = await fetchUser({ username })
+    if (result.userExists) {
+      const userID = result.user.id
+      const snapshot = await badgeModel.get()
+      const userBadges = []
+      snapshot.forEach((item) => {
+        if (item.data().users.includes(userID)) {
+          userBadges.push({ title: item.data().title, description: item.data().description })
+        }
+      })
+      return { userExists: true, userBadges: userBadges }
+    }
+    return { userExists: false, userBadges: [] }
```

Comment:
Can we simply have a default value for `userExists` and `userBadges` declared at let at the top block? Then modify it based on condition and return the values? So we'd have just 1 return statement.

Reply:
Okay makes sense, will do it.